### ISSUE TITLE
Contact

### ISSUE THREAD

**Issue by andresbase** [KyleSanderson/Counter-Strike--Source-changes#1]
Hi @KyleSanderson I tried to find a way to contact you but couldn't find any. Regarding: 
<img width="774" alt="Screen Shot 2019-05-06 at 12 03 08" src="https://user-images.githubusercontent.com/1775039/57248610-a063a180-6ff7-11e9-9b85-e95f1deadc18.png">

You can contact me at andres@eff.org https://eff.org/files/2017/06/28/andres.txt
Or on Keybase andres_base

**Comment by andresbase** [comment]
Hi Kyle, I was hoping you would contact me but you have made some other comments and I want to address it as soon as possible. 

![Screen Shot 2019-05-07 at 09 33 42](https://user-images.githubusercontent.com/1775039/57318340-f7787d80-70ae-11e9-98cb-9b36a1eaaac8.png)

![Screen Shot 2019-05-07 at 09 32 58](https://user-images.githubusercontent.com/1775039/57318348-fe9f8b80-70ae-11e9-8874-e240105433bc.png)

<img width="780" alt="Screen Shot 2019-05-06 at 12 02 11" src="https://user-images.githubusercontent.com/1775039/57323399-9656a700-70ba-11e9-8201-1aae16b9e148.png">


Some of this comments come across as deameaning of the work the community has put. Often they are disruptive to the conversations which we want to be constructive and respectful. We have a Code of Conduct (CoC) to serve as a guide on what our expectations are: https://www.eff.org/pages/eppcode

Specifically:

> * We will exclude you from interaction if you insult, demean, or harass anyone. That is not welcome behavior. We interpret the term “harassment” as including but not limited to disparaging comments about levels of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristics; deliberate intimidation; stalking; sustained disruption of conversations, and unwelcome sexual attention. In particular, we won’t tolerate behavior that excludes participants on the basis of the above characteristics. Private harassment is also unacceptable.
> * Please be kind and courteous. Please take care not to be, or appear to be, mean or rude. Keeping this project a welcoming place is a shared responsibility involving each of us.

It has been explained before in the issues that the redesign took a long time to do with the input and feedback from the community. This is not something we just came up with overnight and deployed. We welcome diverse constructive opinions, but we ask for respect of others opinions and implementations as well.

From the CoC:
> * Respect that people have differences of opinion and that every software design or implementation choice will have different trade-offs, benefits, and costs. When it comes to software design, there is seldom a right answer.

In accordance with our CoC I have proceeded to remove the disruptive comments that do not provide constructive feedback. I have left the comments that had valuable feedback about improvements we can make. 

I invite you to participate in our community in a kind and courteous way.  The steps we take are also explained in the CoC:

> 1. Remarks that violate standards of good conduct, including hateful, hurtful, or exclusionary remarks are not allowed. (Cursing is allowed, but never targeting another user, and never in a hateful manner.)
> 2. Remarks that moderators find inappropriate, whether listed in the code of conduct or not, are also not allowed.
> 3. Moderators will generally first respond to such remarks with a warning. (Egregious harassment may be responded to without a warning.)
> 4. If the warning is unheeded, the user will be “kicked,” i.e., kicked out of the communication channel to cool off.
> 5. If the user comes back and continues to make trouble, they will be banned, i.e., indefinitely excluded.
> 6. Moderators may choose at their discretion to un-ban the user if it was a first offense.
> 7. If a moderator bans someone and you think it was unjustified, please take it up with that moderator, or with the EFF Code of Conduct team, in private. Complaints about bans in-channel are not allowed. Any significant changes to decisions about bans will be reported to the channel by the moderator.
> 8. Moderators are held to a higher standard than other community members. If a moderator creates an inappropriate situation, they should expect less leeway than others.

**Comment by KyleSanderson** [comment]
@andresbase I get it, honestly. The problem is this was my first experience to what the EFF is nearly a decade ago when I tested this on Firefox. HTTPS Everywhere was the first extension I installed when I flipped to Chrome, as an end user. To see the product fall from grace by being given absolutely no oversight to absolutely disastrous changes hurts not only my views on the product but spans the entire globe and I'm sure I'm not alone. Even a little project or release management would have avoided this train wreck and it's absolutely heartbreaking.

I hope to see a positive and transformative change happen within the EFF where people can learn from these mistakes and to strive for higher quality releases in the future.
